GuitarLSTM/LSTM.ipynb

```diff
@@ -1,235 +1 @@
nPPvss5s+fDwB0nh3AXue0srISCQkJPfbB3xcQEDDoGCl5IU730EMP4cSJE9i9e7fQobic0tJSPPLII9iyZQs0Go3Q4bg0i8WCtLQ0/OMf/wAATJgwASdOnMBbb72FBQsWCByd6/j888+xfv16fPLJJxg1ahSysrKwbNkyREZG0nl2Y9RtdBHBwcFQKBQ9RmNUVVUhPDxcoKika+nSpdi4cSO2b9+O6Oho2+3h4eEwGAxobGzstn3X8xweHt7r68DfR7huoerqakycOBFKpRJKpRI7d+7Ea6+9BqVSibCwMDrPdhIREYGRI0d2u23EiBEoKSkB0Hmu+rp2hIeHo7q6utv9JpMJ9fX1dK6t/vKXv2DlypW48847MWbMGNxzzz149NFHsXbtWgB0nh3BXufUkdcSSl4uQqVSYdKkSdi6davtNovFgq1btyIjI0PAyKSFMYalS5fim2++wbZt23o0JU6aNAkeHh7dznNeXh5KSkps5zkjIwPHjx/v9gezZcsWaLXaHh8i7uqKK67A8ePHkZWVZftJS0vD/Pnzbf+n82wfl1xySY/h/vn5+YiLiwMAJCQkIDw8vNu5bmpqwoEDB7qd68bGRmRmZtq22bZtGywWC9LT053wLMSvra0Ncnn3jyqFQgGLxQKAzrMj2OucZmRkYNeuXTAajbZttmzZgpSUlCF1GQGgodL9sWHDBqZWq9mHH37ITp06xZYsWcL8/f27jcYgffvjH//I/Pz82I4dO1hFRYXtp62tzbbNAw88wGJjY9m2bdvY4cOHWUZGBsvIyLDdzw/hveqqq1hWVhbbvHkzCwkJoSG8F9F1tBFjdJ7t5eDBg0ypVLJnn32WFRQUsPXr1zMvLy/28ccf27Z57rnnmL+/P/vuu+/YsWPH2A033NDrcNMJEyawAwcOsN27d7Pk5GS3HsJ7vgULFrCoqCjbUOmvv/6aBQcHs7/+9a+2beg8D1xzczM7evQoO3r0KAPAXnrpJXb06FF29uxZxph9zmljYyMLCwtj99xzDztx4gTbsGED8/LyoqHSzvSvf/2LxcbGMpVKxaZMmcL2798vdEiSAqDXnw8++MC2TXt7O3vwwQdZQEAA8/LyYjfddBOrqKjotp/i4mJ29dVXM09PTxYcHMwee+wxZjQanfxspOX85IXOs/388MMPbPTo0UytVrPU1FT29ttvd7vfYrGwJ598koWFhTG1Ws2uuOIKlpeX122buro6dtdddzEfHx+m1WrZwoULWXNzszOfhqg1NTWxRx55hMXGxjKNRsMSExPZE0880W34LZ3ngdu+fXuv1+QFCxYwxux3TrOzs9n06dOZWq1mUVFR7LnnnrNL/DLGukxTSAghhBAiclTzQgghhBBJoeSFEEIIIZJCyQshhBBCJIWSF0IIIYRICiUvhBBCCJEUSl4IIYQQIimUvBBCCCFEUih5IYQQQoikUPJCCCGEEEmh5IUQQgghkkLJCyGEEEIk5f8BZdYCEH4oew8AAAAASUVORK5CYII=","text/plain":["<Figure size 640x480 with 1 Axes>"]},"metadata":{},"output_type":"display_data"}],"source":["# Compare predict and target\n","import matplotlib.pyplot as plt\n","\n","# sample first 1000 audio data\n","y_test_pred = output[0, :, 0][:1000]\n","y_test_true = y_test[0, :, 0][:1000]\n","plt.plot(range(len(y_test_pred)), y_test_pred, label='pred')\n","plt.plot(range(len(y_test_true)), y_test_true, label='target')\n","plt.legend()"]},{"cell_type":"code","execution_count":24,"metadata":{"executionInfo":{"elapsed":423,"status":"ok","timestamp":1732694136418,"user":{"displayName":"Hua-Hsing Huang","userId":"01596834180958819638"},"user_tz":480},"id":"os6Sy-TrL4Eb"},"outputs":[],"source":["import torchaudio\n","\n","filepath = \"result/ht1_target_predict.wav\"\n","output1 = output[0]\n","torchaudio.save(filepath, output1, sample_rate=44100, channels_first=False)"]},{"cell_type":"code","execution_count":14,"metadata":{"executionInfo":{"elapsed":4,"status":"ok","timestamp":1732693779882,"user":{"displayName":"Hua-Hsing Huang","userId":"01596834180958819638"},"user_tz":480},"id":"C5oGZ6d5eedt"},"outputs":[],"source":[]}],"metadata":{"colab":{"provenance":[]},"kernelspec":{"display_name":"Python 3","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.9"}},"nbformat":4,"nbformat_minor":0}
```

Comment:
When creating the graph for comparison between predicted and target I noticed you took only one channel and compared it rather than both channels, this could possibly be our issue? very strange how our predicted output takes the form of a big sine wave though